vaadin-dir-helper.html

```diff
@@ -0,0 +1,97 @@
+<!--
+@license
+Copyright (c) 2020 Vaadin Ltd.
+This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
+-->
+
+<script>
+  window.Vaadin = window.Vaadin || {};
+
+  Vaadin.DirHelper = class VaadinDirHelper {
+    /**
+     * Get the scroll type in the current browser view.
+     *
+     * @return {String} the scroll type. Possible values are `default|reverse|negative`
+     */
+    static _detectScrollType() {
```

Comment:
You don't really need a class here, neither `static`. Helper can be a plain object with a method.

Also, this method should be considered "public" (called from outside) and not underscored.
Let's change this to be aligned with [Vaadin.FocusablesHelper](https://github.com/vaadin/vaadin-overlay/blob/master/src/vaadin-focusables-helper.html) approach.

Reply:
Updated.